shared/main.lua

```diff
@@ -12,4 +11,18 @@ for _, v in pairs(qbShared.Vehicles) do
 	qbShared.VehicleHashes[v.hash] = v
 end
 
+local weaponConfig = require '@ox_inventory.data.weapons'
+qbShared.Weapons = weaponConfig.Weapons
+qbShared.Components = weaponConfig.Components
+qbShared.AmmoTypes = weaponConfig.Ammo
+
+---@type table<number, Weapon>
+qbShared.WeaponHashes = {}
```

Comment:
This looks like a new table which represents the QB format, but doesn't qbShared.Weapons need to represent the QB format? How are we going to be using these new fields of qbShared?

Reply:
Also, changing qbShared at all could be out of scope for this PR if you want to split if off.

Reply:
the idea was to just make the shared/items.lua be a one way input into ox_inventory format and then we create the qb format for the backwards compatibility.  I hadn't finished the weapons piece which was going to be mostly the stuff from #451